packages/core/src/lib/nodeInfo.ts

```diff
@@ -1,13 +1,13 @@
 export type nodeInfo = typeof swankyNode;
 
 export const swankyNode = {
-  version: "0.10.0",
-  polkadotPalletVersions: "polkadot-v0.9.27",
-  supportedInk: "v3.3.1",
+  version: "1.0.0",
+  polkadotPalletVersions: "polkadot-v0.9.33",
+  supportedInk: "v4.0.0",
```

Comment:
I think if we update supported ink version, better to update templates at the same time as well. Although I assume we can deploy ink 3.x.x contracts to swanky-node 1.0.0 (technically not all, some doesn't have backward compatibility).
Plus, we have UI issue, polkadot.js is available for WeightV1 node but contracts-ui is not available for WeightV2 swanky-node 1.0.0 due to instant finalization issue. So, I suggest to update local node with a separated PR at the right timing.

Reply:
Also, for weightV2 node, we need to construct gasLimit like this https://github.com/polkadot-js/api/issues/5255#issuecomment-1372540513